dynatrace_topology/stackstate_checks/dynatrace_topology/dynatrace_topology.py

```diff
@@ -8,30 +8,48 @@
 from schematics.types import IntType, URLType, StringType, ListType, BooleanType, ModelType, DictType
 
 from stackstate_checks.base import AgentCheck, StackPackInstance, HealthStream, HealthStreamUrn, Health
-from stackstate_checks.dynatrace.dynatrance_client import DynatraceClient
+from stackstate_checks.dynatrace.dynatrace_client import DynatraceClient
 from stackstate_checks.utils.identifiers import Identifiers
 
+from stackstate_checks.dynatrace.entity_data_types import ApplicationEntity, HostEntity, QueueEntity, ServiceEntity, ProcessGroupEntity, ProcessGroupInstanceEntity, CustomDeviceEntity
+
 VERIFY_HTTPS = True
 TIMEOUT = 10
 RELATIVE_TIME = 'hour'
 ENVIRONMENT = 'production'
 DOMAIN = 'dynatrace'
-CUSTOM_DEVICE_DEFAULT_RELATIVE_TIME = '1h'
-CUSTOM_DEVICE_DEFAULT_FIELDS = '+fromRelationships,+toRelationships,+tags,+managementZones,+properties.dnsNames,' \
-                               '+properties.ipAddress'
-
-TOPOLOGY_API_ENDPOINTS = {
-    "process": "api/v1/entity/infrastructure/processes",
-    "host": "api/v1/entity/infrastructure/hosts",
-    "application": "api/v1/entity/applications",
-    "process-group": "api/v1/entity/infrastructure/process-groups",
-    "service": "api/v1/entity/services",
-    "custom-device": "api/v2/entities",
-    "synthetic-monitor": "api/v1/synthetic/monitors"
+
+API_V2_DEFAULT_RELATIVE_TIME = '1h'
+API_V2_DEFAULT_FIELDS_STRING = '+fromRelationships,+toRelationships,+tags,+managementZones,+properties'
+
+TOPOLOGY_API_SPEC = {
```

Comment:
i don't see these changes reflected in the test code, i am expecting the output to have changed somewhat and the mocking needing some work there? I see the tests are failing aswell